portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -3566,8 +3565,7 @@ $.extend(true, doaj, {
                     var result = '<br/><p><a class="readonly_journal_link button" href="';
                     result += params.readOnlyJournalUrl;
                     result += resultobj.admin.current_journal;
-                    result += '" target="_blank"';
-                    result += '>View journal being updated</a></p>';
+                    result += '">View journal being updated</a></p>';
```

Comment:
Are we sure about not opening these pages in new windows?  I personally think it's great to get rid of this, but it was there because it was wanted that way, are editorial happy with the change?

Reply:
Idem